src/style.css

```diff
@@ -2,2 +1,71 @@
+* {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-index {
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  padding: 0;
+  margin-right: -20px;
+}
+
+.nav-item {
+  display: inline-block;
+  margin-right: 20px;
+}
+
+.nav-link {
+  text-transform: uppercase;
+  line-height: 60px;
+  text-decoration: none;
+  color: black;
+  font-weight: 500;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  position: relative;
+  font-style: normal;
+}
+
+.color{
+  color: #00ACDC;
+}
+
+.is.active {
```

Comment:
The selector `.is.active` seems incorrect. If you intend to target elements with both `is` and `active` classes, there should be no space between them. However, if you meant to target elements with both classes separately, use `.is .active` with a space.